{
    cosvxx: 2.0,
    cosvyy: 2.0,
    cosxx: 0.7,
    cosyy: 0.7,
    process_noise_level: 0.34,
    use_process_noise: true
}
